src/routes/contacts/+page.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+  import LabelInput from '$lib/component/LabelInput.svelte';
+  import Graph from './Graph.svelte';
+  import Icon from 'svelte-icons-pack/Icon.svelte';
+  import BiMap from 'svelte-icons-pack/bi/BiMap';
```

Comment:
please use the `Icons`  pattern we used for the footer. Add the export in the `Icons.ts` and use it in all icons' sources